kernels/cclo/fw/sw_apps/ccl_offload_control/src/ccl_offload_control.h

```diff
@@ -33,7 +33,7 @@
 //Datapath width (in bytes) for all streams everywhere in the design
 #define DATAPATH_WIDTH_BYTES 64
 
-//AXIS interfaces to/from MB 
```

Comment:
no functional changes in this file that i can tell, please revert it to the version in dev so we shrink the size of the PR